composer.json

```
@@ -11,6 +11,7 @@
         "nette/robot-loader": "^3.2",
         "nette/utils": "^3.1",
         "nikic/php-parser": "4.5",
+        "sebastian/diff": "^4.0",
```

```suggestion
        "sebastian/diff": "^3.0|^4.0",
```

Test on PHP 7.2 are failing, this should help